src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let calls = 0;
+
+  const getSum = function(firstNum) {
+    calls++;
+
+    return secondNum => {
+      if (calls >= 4 && calls % 2 === 0) {
+        return 'Bzzz... Error!';
+      } else {
+        return firstNum + secondNum;
+      }
```

Comment:
```suggestion
      }
      
      return firstNum + secondNum;
```
This `else` is redundant here